nbbuild/licenses/names.properties

```diff
@@ -19,9 +19,9 @@ GPL-2-CP=GPL v2 with Classpath exception (http://openjdk.java.net/legal/gplv2+ce
 Apache-2.0=Apache Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt)
 Apache-2.0-groovy=Apache Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt) with Apache Groovy addendums
 Apache-2.0-lucene=Apache Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt) with Apache Lucene addendums
-Apache-2.0-spring3.2.7=Apache Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt) with Spring addendums, version 3.2.7
-Apache-2.0-spring4.0.1=Apache Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt) with Spring addendums, version 4.0.1
-Apache-2.0-spring5.2.2=Apache Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt) with Spring addendums, version 5.2.2
+Apache-2.0-spring3.2.18=Apache Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt) with Spring addendums, version 3.2.18
+Apache-2.0-spring4.3.29=Apache Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt) with Spring addendums, version 4.3.29
+Apache-2.0-spring5.2.9=Apache Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt) with Spring addendums, version 5.2.9
```

Comment:
I would simplify these to (that needs updates at the use-sites though):

- Apache-2.0-spring3
- Apache-2.0-spring4
- Apache-2.0-spring5

and change the versions at the end to "3.X", "4.X" and "5.X".
-

Reply:
I agree with your suggestion. Made the changes and commit.

Reply:
can you check too why this .cache/netbeans/12.1/executor-snippets/run.xml:24: is happen when you try compile a swing program ?